docs/how-to/how-to-run-kata-containers-with-SE-VMs.md

```diff
@@ -0,0 +1,387 @@
+
+In this section, we will specifically guide you on building a payload image
+(i.e., `kata-deploy`) for confidential containers. For the remaining instructions,
+please refer to the
+[documentation](https://github.com/confidential-containers/operator/blob/main/docs/how-to/INSTALL-CC-WITH-IBM-SE.md)
+for confidential containers.
+
+
+```
+$ cd $GOPATH/src/github.com/kata-containers/kata-containers
+$ host_key_document=$HOME/host-key-document/HKD-0000-0000000.crt
+$ mkdir hkd_dir && cp $host_key_document hkd_dir
+$ # kernel and rootfs-initrd are built automactially by the command below
+$ sudo -E PATH=$PATH HKD_PATH=hkd_dir SE_KERNEL_PARAMS="agent.log=debug" \
+make boot-image-se-tarball
+$ sudo -E PATH=$PATH make qemu-tarball
+$ sudo -E PATH=$PATH make virtiofsd-tarball
+$ # shim-v2 should be built after kernel due to dependency
+$ sudo -E PATH=$PATH make shim-v2-tarball
+$ mkdir kata-artifacts
+$ build_dir=$(readlink -f build)
+$ cp -r $build_dir/*.tar.xz kata-artifacts
+$ ls -1 kata-artifacts
+kata-static-boot-image-se.tar.xz
+kata-static-kernel.tar.xz
+kata-static-qemu.tar.xz
+kata-static-rootfs-initrd.tar.xz
+kata-static-shim-v2.tar.xz
+kata-static-virtiofsd.tar.xz
+$ ./tools/packaging/kata-deploy/local-build/kata-deploy-merge-builds.sh kata-artifacts
+```
+
+In production, the environment variables `SIGNING_KEY_CERT_PATH` and
+`INTERMEDIATE_CA_CERT_PATH` should be exported like the manual configuration.
+If a rootfs-image is required for other available runtime classes (e.g. `kata` and `kata-qemu`)
+without the Secure Execution functionality, please run the following command
+before running `kata-deploy-merge-builds.sh`:
+
+```
+$ sudo -E PATH=$PATH make rootfs-image-tarball
+```
+
+At this point, you should have an archive file named `kata-static.tar.xz` at the project root,
+which will be used to build a payload image. If you are using a local container registry at
+`localhost:5000`, proceed with the following:
+
+```
+$ docker run -d -p 5000:5000 --name local-registry registry:2.8.1
+```
+
+Build and push a payload image with the name `localhost:5000/build-kata-deploy` and the tag
+`latest` using the following:
+
+```
+$ sudo -E PATH=$PATH ./tools/packaging/kata-deploy/local-build/kata-deploy-build-and-upload-payload.sh kata-static.tar.xz localhost:5000/build-kata-deploy latest
+... logs ...
+Pushing the image localhost:5000/build-kata-deploy:latest to the registry
+The push refers to repository [localhost:5000/build-kata-deploy]
+76c6644d9790: Layer already exists
+2413aff53bb1: Layer already exists
+91462f44bb06: Layer already exists
+2ad49fac591a: Layer already exists
+5c75aa64ef7a: Layer already exists
+test: digest: sha256:25825c7a4352f75403ee59a683eb122d5518e8ed6a244aacd869e41e2cafd385 size: 1369
+```
+
+## Considerations for CI
+
+If you intend to integrate the aforementioned procedure with a CI system, it is
+recommended to configure the following setup for an environment variable.
```

Comment:
minor: remove "it is recommended" and simply use "configure"

Reply:
Thanks for the feedback. The document has been updated.